aws-amplify/docs
# Add custom analytics event for the "What's New" banner

#### Description of changes:
- Added a custom event for the "Whats New" banner

Staging site: https://banner-analytics.d3kzzkiugubm89.amplifyapp.com/

To test:
1. Go to the staging site
2. Open up dev tools and navigate to the "Omnibug" tab ([link to install Omnibug on your browser](https://omnibug.io/))
3. Click on the "Whats New" banner
4. Look at the click event in the "Omnibug" tab before it "Navigated to https://aws.amazon.com/blogs/mobile/announcing-aws-amplifys-graphql-api-cdk-construct-deploy-real-time-graphql-api-and-data-stack-on-aws/"
5. Verify that in the "Context Data" section, it has the `event.name` property as `WhatsNewBanner`

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
